tests/dragonfly/redis_replicaiton_test.py

```diff
@@ -27,6 +27,35 @@ def stop(self):
         except Exception as e:
             pass
 
+async def await_synced(master_port, replica_port, dbcount=1):
```

Comment:
just add a comment for the logic how you check that redis and dragonfly are synced